buildraw4

```diff
@@ -101,7 +101,9 @@ function install_walla()
     # Skip bro installation if in docker environment, bro in apt repo will be used in Docker
     if [[ ! -f /.dockerenv ]]; then
         echo "Installing bro..." 
-        cd $basedir/imports && wget https://github.com/firewalla/firewalla/releases/download/1.6/bro49.tar.gz -O bro49.tar.gz -q
+        if [[ ! -f $basedir/imports/bro49.tar.gz ]]; then
```

Comment:
md5 needs to be checked before trust this local file, this file may be corrupted if wget exited abnormally last time.